install/inc/schema_mysql.sql

```diff
@@ -1376,8 +1376,8 @@ create table ca_user_roles
    code                           varchar(20)                    not null,
    description                    text                           not null,
    rank                           smallint unsigned              not null default 0,
-   vars                           text                           not null,
-   field_access                   text                           not null,
+   vars                           longtext                       not null,
```

Comment:
Do you think `longtext` (4GB) is overkill?  Would `mediumtext` (16MB) be better?  For reference, `text` is 64kB.